website/pages/en/index.js

```diff
@@ -71,8 +71,8 @@ module.exports = class Index extends React.Component {
         {[
           {
             content:
-              '<p>Richie is a specialized Content Management System designed to make learning portals. It is full-featured and ready-to-go.</p>' +
-              '<p>It is a toolbox to easily create full fledged websites with a catalog of online courses in days, not months.</p>' +
+              "<p>Richie helps educators create online learning portals. </p>" +
+              "<p>Build websites and online course catalogs in days with Richie's content management system.</p>" +
```

Comment:
can we say like this?
```suggestion
              "<p>Build websites around online course catalogs in days with Richie's content management system.</p>" +
```
or
```suggestion
              "<p>Build websites including online course catalogs in days with Richie's content management system.</p>" +
```

Reply:
The second option flows better.

Reply:
:+1: